Makefile

```diff
@@ -7,16 +7,16 @@ R_VERSION := 4.0.4
 docker-build-%:
 	docker build -t rstudio/shinyapps-package-dependencies:$* --build-arg R_VERSION=$(R_VERSION) -f Dockerfile.$* .
 
-test-xenial-%:
+test-xenial-%: test install Dockerfile.xenial
```

Comment:
Can you explain the context for these Makefile changes?

Reply:
I can't. Reverting

Reply:
jk. I can.

From [deaa12](https://github.com/rstudio/shinyapps-package-dependencies/pull/302/commits/deaa121736a985b6a19547ac33debdd3d01783df), I added that if `./test` file, `./install` file, or `Dockerfile.xenial` changed, then it would be nice if the method would run again. 

When running locally, the command wouldn't do anything if it a successful completion. Now, if any of those files changes, it is forced to run again.